app/navigation/nonsipp/LandOrPropertyNavigator.scala

```diff
@@ -42,6 +42,9 @@ object LandOrPropertyNavigator extends JourneyNavigator {
 
     case LandOrPropertyAddressLookupPage(srn, index) =>
       controllers.routes.UnauthorisedController.onPageLoad()
+
+    case WhyDoesSchemeHoldLandPropertyPage(srn, index) =>
```

Comment:
Is there any routing from a previous page to this page?

Reply:
this should call the page (my ticket) is not ready yet

Reply:
Hi George, Yes there is a routing from previous page, but that page is in block now(PSR-205).

Reply:
@westwater , I think it's better to set navigation from address lookup page to this page as of now. Otherwise page is not displaying. Once we get the other page in between we can change the routing accordingly.